products/PurePhone/services/desktop/endpoints/contacts/ContactHelper.cpp

```diff
@@ -244,9 +244,13 @@ namespace sdesktop::endpoints
         auto listener = std::make_unique<db::EndpointListener>(
             [](db::QueryResult *result, Context context) {
                 if (auto contactResult = dynamic_cast<db::query::ContactUpdateResult *>(result)) {
-
-                    context.setResponseStatus(contactResult->getResult() ? http::Code::NoContent
-                                                                         : http::Code::InternalServerError);
+                    if (contactResult->hasDuplicates()) {
+                        context.setResponseStatus(http::Code::Conflict);
```

Comment:
Consult with the Center team if they need to know which number is duplicated.
If they need to add this information to the response